src/ToricVarieties/cohomCalg/VanishingSets/methods.jl

```diff
@@ -0,0 +1,54 @@
+@doc Markdown.doc"""
+    contains(tvs::ToricVanishingSet, l::ToricLineBundle)
+
+Checks if the toric line bundle `l` is contained in the toric vanishing set `tvs`.
+
+# Examples
+```jldoctest
+julia> dP1 = del_pezzo(1)
+A normal, non-affine, smooth, projective, gorenstein, fano, 2-dimensional toric variety without torusfactor
+
+julia> l = ToricLineBundle(dP1, [3,2])
+A toric line bundle on a normal toric variety
+
+julia> all_cohomologies(l)
+3-element Vector{fmpz}:
+ 7
+ 0
+ 0
+
+julia> vs = vanishing_sets(dP1)
+3-element Vector{ToricVanishingSet}:
+ A toric vanishing set for cohomology index 0
+ A toric vanishing set for cohomology index 1
+ A toric vanishing set for cohomology index 2
+
+julia> contains(vs[1], l)
+false
+
+julia> contains(vs[2], l)
+true
+
+julia> contains(vs[3], l)
+true
+```
+"""
+function contains(tvs::ToricVanishingSet, l::ToricLineBundle)
+    if toric_variety(l) !== toric_variety(tvs)
+        throw(ArgumentError("The line bundle must be defined on the same toric variety as the vanishing set."))
+    end
+    if isfull(tvs)
+        return true
+    end
+    class = divisor_class(l).coeff
+    for i in 1:length(shifted_cones(tvs))
+        c = cone(shifted_cones(tvs)[i])
+        pt = class - offset(shifted_cones(tvs)[i])
+        pt = [k for k in pt][1,:]
```

Comment:
Huh, what's going on in that line?

Reply:
Looks surprising, achieves the following:
(1) In 46, `typeof(pt)` is `fmpz_mat`.
(2) In 47, `typeof(pt)` is `Vector{fmpz}`.
The latter is necessary (at least as of now) in the next line, where I check if the point `pt` is contained in the cone `c`:
```julia
if Polymake.polytope.contains(c.pm_cone, pt)
```
But maybe there is a better way?

Reply:
https://github.com/oscar-system/Oscar.jl/pull/1269 contains a better method for checking if a point is contained in a cone. Once this is merged, the above lines can likely be improved. I reserve this for a future PR. For now, added this also to the issue in https://github.com/oscar-system/Oscar.jl/issues/1257.